### PR TITLE
Remove the front-end

### PR DESCRIPTION
**Pull request by openjck**

### CODE REVIEW

**Comment by openjck** [comment]
@robhudson and @spasovski: Please let me know if I've missed anything.

@robhudson: I'm not exactly sure what to do about authentication. Visiting the API endpoints currently loads 403 errors. Let me know what you think should be done and/or feel free to update the PR. :smiley:

File: bin/run [hunk 1]
```diff
@@ -32,21 +27,12 @@ case $1 in
...
-            run_back_end_tests
-            run_front_end_tests
-            if [[ ! -z ${CI} ]]; then
-                coverage xml
-                bash <(curl -s https://codecov.io/bash) -s /tmp
-            else
-                coverage report -m
-            fi
+            coverage report -m
         fi
```
Comment (robhudson) [inline bin/run]: 👍

**Comment by spasovski** [comment]
Looks good from here too. Thanks for the quick turnaround.

**Comment by openjck** [comment]
> These changes look good to me. About authentication and the API... it was like this before. You have to get a Django session cookie and then send that with the API requests. So I think there's no changes to be done here wrt auth.

Ah. How does one get that session cookie?

**Comment by openjck** [comment]
I just want to be extra sure that this still works.

**Comment by openjck** [comment]
Do we need Whitenoise if we're not hosting front-end assets?

**Comment by openjck** [comment]
@robhudson I think I figured out what's going on with auth. When you get a chance, double-check that the API still works as expected. :smiley:

**Comment by robhudson** [comment]
> 
> 
> Do we need Whitenoise if we're not hosting front-end assets?

No, we don't.

**Comment by openjck** [comment]
> No, we don't.

Can I delegate that to you since you'll be doing other back-end tweaks? :smiley:

**Comment by robhudson** [comment]
> Can I delegate that to you since you'll be doing other back-end tweaks? 😃

Yes of course, I'm on it already. 👍

**Comment by robhudson** [comment]
@openjck Can you give this a final look now that I've added the backend pieces?

**Comment by openjck** [comment]
Absolutely. I'll have to get to this next week, as I'm on PTO tomorrow. Thanks for the updates!

File: bin/run [hunk 2]
```diff
@@ -6,11 +6,6 @@ run_back_end_tests() {
     coverage run ./manage.py test
 }
 
-# TODO
-#run_front_end_tests() {
-    # ./node_modules/gulp/bin/gulp.js test
-#}
-
```
Comment (openjck) [inline bin/run]: We could probably remove this altogether, right?

**Comment by openjck** [comment]
This looks good to me, but I'm not sure how to use the backend to verify that it works correctly. I'll let you decide if you think we need docs for that, but for now at least, how would one use */v2/enrollment*?

File: bin/run (hunk 2 above)
Comment (robhudson) [reply bin/run]: The file? We still need the tests to run for the couple tests that are left.

**Comment by robhudson** [comment]
> This looks good to me, but I'm not sure how to use the backend to verify that it works correctly. I'll let you decide if you think we need docs for that, but for now at least, how would one use _/v2/enrollment_?

The tests are a good source of how this is expected to be used:
https://github.com/mozilla/firefox-test-tube/blob/remove-front-end/tests/test_api.py

This is only supposed to be called internally and it's already set up and running so I think the tests are good enough. Nothing else should be calling this API endpoint.

**Comment by openjck** [comment]
Do we still need *backend/templates/viewer/index.html*?

**Comment by openjck** [comment]
This gets an r+ from me, pending the question about *backend/templates/viewer/index.html*.

**Comment by robhudson** [comment]
> 
> 
> Do we still need _backend/templates/viewer/index.html_?

I'm not seeing that file in the current branch.

**Comment by openjck** [comment]
> I'm not seeing that file in the current branch.

Ah. That's right. I got mixed up. Okay! Merging...

**Comment by robhudson** [comment]
@sunahsuh FYI. After this branch merges the only thing in this repo will be the import script and a single API endpoint to ingest the telemetry-streaming data.

**Pull request merged**